samples/sampler/stories/default/Popup.js

```diff
@@ -31,15 +31,17 @@ storiesOf('Agate', module)
 					<Popup
 						centered={boolean('centered', Config)}
 						closeButton={boolean('closeButton', Config)}
+						duration={select('duration', ['short', 'medium', 'long'], Config)}
 						noAnimation={boolean('noAnimation', Config)}
 						noAutoDismiss={boolean('noAutoDismiss', Config)}
 						onClose={action('onClose')}
 						onHide={action('onHide')}
 						open={boolean('open', Config)}
-						position={select('position', ['center', 'top'], Config)}
+						position={select('position', ['bottom', 'center', 'fullscreen', 'left', 'right', 'top'], Config)}
 						scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config, 'translucent')}
 						spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config, 'self-only')}
 						title={text('title', Config, 'Title')}
+						type={select('type', [null, 'fade', 'slide'], Config)}
```

Comment:
Do we need `null` here?

Reply:
Yes, for when `type` is not specified and to apply this:
transitionType: ({position, type}) => {
			const setTransitionType = position === 'center' ? 'fade' : 'slide';

			return type ? type : setTransitionType;
		}

Reply:
I see. Then I think we should remove default props for type in jsdoc.